ui/components/MesheryPerformanceComponent.js

```diff
@@ -391,6 +375,21 @@ class MesheryPerformanceComponent extends React.Component {
     }, () => {});
   }
 
+  getSMPMeshes = () => {
+    const self = this
+    dataFetch('/api/perf/meshes', {
+      credentials: 'same-origin',
```

Comment:
Specify method here

Reply:
Just to clarify, should I covert the dataFetch function options to:
```diff
credentials: 'same-origin',
credentials: 'include',
+ method: 'GET'
```

Reply:
Yes